CppCoreGuidelines.md

```diff
@@ -8339,7 +8339,7 @@ Unless you really need pointer semantics, use a local resource object:
 		// ...
 	}
 
-**Note**: If there is no obvious resource handle, cleanup actions can be represented by a [`Finally` object](#Re-finally)
```

Comment:
This should actually be a [`finally_action` object].  Can you update your patch to reflect that?
Thanks!